src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderTypeAheadMenu.tsx

```diff
@@ -85,6 +94,15 @@ export class GlobalHeaderTypeAheadMenu extends React.Component<Props, State> {
     }
   }
 
+  private sendTypeAheadSearchEvent = (e: ChangeEvent<HTMLInputElement>) => {
+    const {typeAheadEventPrefix} = this.props
+
+    // No event should be sent if the input field is empty, or just contains whitespace.
+    if (e.target.value.trim().length) {
```

Comment:
apart from not using `e` to represent events mentioned previously, does this need to use optional chaining? If no event is sent if the input field is empty, will trying to `trim` its value end up casing a null pointer exception?

Reply:
Tested this with empty fields (which seem to send empty string), but no harm in adding optional chaining here, in case a null could be sent.